### PR TITLE
feat: added onPaste support

### PR DESCRIPTION
**Pull request by kornasevich**

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -44,6 +44,9 @@ export default function App() {
         placeholder={'placeholder'}
         style={styles.editor}
         toolbarRef={toolbarRef}
+        onPaste={() => {
+          console.log('onPaste');
```
Comment (gontovnik) [inline example/src/App.tsx:48]: Why console log?

File: example/package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "react-dom": "18.2.0",
     "react-native": "0.71.7",
     "react-native-web": "~0.18.10",
-    "react-native-webview": "^12.0.2"
+    "react-native-webview": "11.26.0"
```
Comment (gontovnik) [inline example/package.json:18]: Why downgrade?

**Review by gontovnik — CHANGES_REQUESTED** [review]

File: example/package.json (hunk 2 above)
Comment (kornasevich) [reply example/package.json:18]: because the error version mismatch and RTE does not work

File: example/src/App.tsx (hunk 1 above)
Comment (kornasevich) [reply example/src/App.tsx:48]: as example

**Review by gontovnik — APPROVED** [review]

**Pull request merged**